client/src/Components/PrivateRoute/PrivateRoute.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import { withRouter, Route, Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+
+import HomeView from "../HomeView/HomeView";
+
+class PrivateRoute extends Component {
+  render() {
+    let protectedComponent = props => {
```

Comment:
I don't think this needs to be a function. The render method already is a function and you have access to props